.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "3.6.4"
+cache: pip3
+services:
+  - postgresql
+install:
+  - pip3 install -r requirements.txt
+  - pip3 install pytest pytest-cov
+  - pip3 install coveralls
+before_script:
+  - psql -c "CREATE DATABASE tbc_db;" -U postgres
+  - psql -c "CREATE USER tbc_db WITH PASSWORD '12345678';" -U postgres
+script:
+  - pytest test --cov=test/
```

Comment:
Good work so far. However, this line `pytest test --cov=test` will return your coverage for functionality in the package test which I think is not what you might be looking for. I would recommend you consider reporting the coverage for the entire application. in this case you would replace `test` with `app`

Reply:
Yes, that's so true. I set coverage to the test directory when i was still trying to figure out what coveralls are.